impress-org/givewp
# Fix: Legacy template now displays correct goal amount

Resolves #6337

## Description

This PR resolves the bug with the incorrect goal amount displayed when using the Legacy template. The bug is resolved by getting the correct form earnings. 


## Testing Instructions

1. Install fee recovery add-on
2. Create a new donation form ( legacy template ) that has the Donation Goal and Fee Recovery  option enabled
3. Make a donation, but make sure you opted in to cover the fees
4. Navigate back to the donation form page and check the raised amount - the value should include the fees. 



## Pre-review Checklist
